src/component/helper/MapDraw.js

```diff
@@ -245,22 +237,33 @@ MapDraw.prototype = {
                 }
             }
 
+            var transformPoint = function (point) {
+                return [
+                    point[0] * scale[0] + geo.position[0],
+                    point[1] * scale[1] + geo.position[1]
+                ];
+            };
+
             zrUtil.each(region.geometries, function (geometry) {
                 if (geometry.type !== 'polygon') {
                     return;
                 }
                 compoundPath.shape.paths.push(new graphic.Polygon({
                     segmentIgnoreThreshold: 1,
                     shape: {
-                        points: geometry.exterior
+                        points: zrUtil.map(geometry.exterior, function (point) {
+                            return transformPoint(point);
+                        })
```

Comment:
Use naive `for` loop may have better performance. Especially on the large GeoJSON.